static/scss/auth.scss

```diff
@@ -15,15 +15,18 @@
 .auth-page {
   @include breakpoint(mobile) {
     width: 95%;
-    margin: auto;
 
-    form,
-    .actions,
-    .card-contents {
-      width: 100%;
+    .main-content {
+      margin: auto;
+      max-width: 470px;
     }
   }
 
+  form {
+    max-width: 350px;
```

Comment:
I guess this is a nitpick, but I think the width should be fixed on `.card-contents`, not `form`. The 350px width is defined for `.card-contents` when the window is >800px wide. I think it should just be set to that width for any window width above mobile. That way you wouldn't have the `.card-contents` div suddenly growing to ~450px wide when the screen width drops below 800px. I don't think there's any real reason for that

![ss 2018-09-27 at 17 27 24](https://user-images.githubusercontent.com/14932219/46175779-1bebbc00-c27b-11e8-851c-b058a57fb25e.png)

Reply:
are you seeing the card contents div growing super wide like that? I can't repro that on my machine. The reason I set the the max-width on `form` was the mobile display, if I set the `max-width` on `.card-contents` instead it breaks the layout with flexbox:

![max-width-on-card](https://user-images.githubusercontent.com/6207644/46212308-ac6ddf00-c303-11e8-9add-71961445b258.png)

there may be another way around it, but setting the max-width on the form and actions was the simplest way I found.

Reply:
so I can refactor it to set `max-width: 350px;` on `.card-contents` at all widths, and as long as I keep `margin: auto;` on at mobile the `width: 100%;` on `.form` and `.actions` will preserve the layout - I can do that if you think it's more logical.

Reply:
@aliceriot can you right-align the "next" button with the "email" box ?

Reply:
yeah, that's pretty simple